coeus-impl/src/main/java/org/kuali/kra/subawardReporting/printing/xmlstream/SubAwardFDPPrintXmlStream.java

```diff
@@ -325,16 +334,20 @@ public void setSubcontractAmountInfo(SubContractData subContractData, SubAward s
                 subContractAmountInfo.setObligatedAmount(subaward.getTotalObligatedAmount().bigDecimalValue());
                 subContractAmountInfo.setAnticipatedAmount(subaward.getTotalAnticipatedAmount().bigDecimalValue());
             }
-            if(subaward.getSubAwardAmountInfoList() != null && !subaward.getSubAwardAmountInfoList().isEmpty()){
-                SubAwardAmountInfo amountInfo = subaward.getSubAwardAmountInfoList().get(subaward.getSubAwardAmountInfoList().size() - 1);
+            if(subaward.getAllSubAwardAmountInfos() != null && !subaward.getAllSubAwardAmountInfos().isEmpty()){
+                SubAwardAmountInfo amountInfo = subaward.getAllSubAwardAmountInfos().get(subaward.getAllSubAwardAmountInfos().size() - 1);
                 subContractAmountInfo.setObligatedChange(amountInfo.getObligatedChange().bigDecimalValue());
                 subContractAmountInfo.setAnticipatedChange(amountInfo.getAnticipatedChange().bigDecimalValue());
 
                 if(amountInfo.getPeriodofPerformanceStartDate() != null){
-                    subContractAmountInfo.setPerformanceStartDate(getDateTimeService().getCalendar(amountInfo.getPeriodofPerformanceStartDate()));
+                    subContractAmountInfo.setPerformanceStartDate(getDateTimeService().getCalendar(subaward.getStartDate()));
```

Comment:
I do not understand why you check amountInfo.getPeriodofPerformanceStartDate() != null but then continue to use subaward.getStartDate(). Which one should you be using?